src/cov.jl

```diff
@@ -29,6 +29,21 @@ scattermat_zm(x::DenseMatrix, wv::WeightVec, vardim::Int) =
     _symmetrize!(Base.unscaled_covzm(x, _scalevars(x, values(wv), vardim), vardim))
 
 if VERSION < v"0.5.0-dev+679"
+"""
+    scattermat(x, [wv,] [mean=nothing,] [vardim=1]) -> Matrix
```

Comment:
Add `;` before keyword arguments. Also, they don't need brackets since the default value already shows that they are optional arguments.